src/main/kotlin/com/aol/mobile/sdk/apitracker/TrackerPlugin.kt

```diff
@@ -99,9 +99,15 @@ class TrackerPlugin : Plugin<Project> {
                             changeReportFile = File(buildDir, "changeReport.txt")
                         }
                     }.dependsOn(getByName("kaptReleaseKotlin"))
+
+                    create("generateProguard", ProguardGenerateTask::class.java) { task ->
```

Comment:
This will not work properly, this task must be executed prior proguard processing

Reply:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html

Reply:
Need one more dependency